src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobBuilder.Enumerators.cs

```diff
@@ -28,7 +28,7 @@ internal Chunks(BlobBuilder builder)
             }
 
             object IEnumerator.Current => Current;
-            public BlobBuilder Current => _currentOpt;
```

Comment:
For me it should be nullable as it is a possible that `Current` return null.

Reply:
That's true, however this is only the case when calling `Current` _before_ the any call to `MoveNext()` has been made. The problem with accurately annotating most `IEnumerator` implementations is that it forces us to also annotate the corresponding `IEnumerable` as yielding nullable elements, which is not necessarily the case. So we generally tolerate slightly inaccurate IEnumerator annotations, particularly when they are internal implementations.